packages/common/src/server.ts

```diff
@@ -93,6 +93,7 @@ export type YogaServerOptions<
           ? [serverContext?: TServerContext | undefined]
           : [serverContext: TServerContext]
       ) => GraphiQLOptions | false)
```

Comment:
Good catch! I think it is fine to accept `true`.

Reply:
Not sure if you want me to change something, but the code doesn't like that, as you can see in this spec:

```
createServer({
  // @ts-expect-error This is not allowed:
  graphiql: true,
})
```

Which makes the documentation a bit weird, where it says 
```
  /**
   * GraphiQL options
   *
   * Default: `true`
   */
```

What actually happens is default config becomes `() => ({})` which is truthy and the graphiql gets mounted with defaults. So the only boolean option you may use is today `false`.